frontend/src/App.vue

```diff
@@ -54,11 +56,19 @@ export default {
       return this.$route.name === 'Confirmation';
     },
   },
+  methods: {
+    ...mapActions({
+      sync: SYNC,
+      fetchUser: FETCH_USER,
+    }),
+  },
   created() {
     if (this.language) {
       /* eslint-disable-next-line no-underscore-dangle */
       this._i18n.locale = this.language;
     }
+    this.sync();
```

Comment:
`this[SYNC]()`?

Reply:
nope. line 61 maps action to a name. in this case it's called sync